available_services/weatherservice.yml

```diff
@@ -15,17 +14,24 @@ services:
   weathersrv:
     image: ghcr.io/openagri-eu/weather-service:latest
     depends_on:
-      - mongodb
+      - weathersrv_db
+      - gatekeeper
+    restart: on-failure
     ports:
-      - '8000:8000'
       - "${WEATHER_SRV_PORT}:${WEATHER_SRV_PORT}"
     environment:
       LOGGING_LEVEL: ${WEATHER_SRV_LOGGING_LEVEL:-INFO}
       WEATHER_SRV_PORT: ${WEATHER_SRV_PORT:-8000}
+      WEATHER_SRV_HOSTNAME: ${WEATHER_SRV_HOSTNAME:-weathersrv}
       DATABASE_URI: ${WEATHER_SRV_DATABASE_URI}
       DATABASE_NAME: ${WEATHER_SRV_DATABASE_NAME}
+      GATEKEEPER_APP_PORT: ${GATEKEEPER_APP_PORT}
+      # If the above value is set then weather service will try to integrate with Gatekeeper
+      GATEKEEPER_URL: ${GATEKEEPER_URL:-http://gatekeeper}
+      GATEKEEPER_SUPERUSER_USERNAME: ${GATEKEEPER_SUPERUSER_USERNAME}
```

Comment:
humm, i think it would be best to change this internal env var name to something like:
`WEATHER_SRV_GATEKEEPER_USER` and  `WEATHER_SRV_GATEKEEPER_PASSWORD`.
For now we can use the GATEKEEPER_SUPERUSER_USERNAME as the value for this, but the idea is that in the future each service will have their own user/pass.  
But again, for now you can have this to something like:
`WEATHER_SRV_GATEKEEPER_USER=${GATEKEEPER_SUPERUSER_USERNAME}` 
`WEATHER_SRV_GATEKEEPER_PASSWORD=${GATEKEEPER_SUPERUSER_PASSWORD}` 

@pranavnbapat, what do you think?

Reply:
yes I agree sounds more self-explanatory